### PR TITLE
Cache which capabilities are enabled in the context (fixes #20534)

### LINKED ISSUE #20534: Cache which server-side GL capabilities are enabled
https://www.khronos.org/registry/OpenGL-Refpages/es2.0/xhtml/glEnable.xml
https://www.khronos.org/registry/OpenGL-Refpages/es2.0/xhtml/glIsEnabled.xml
https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.3

### PR DESCRIPTION
**Pull request by nox**
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21246)

### CODE REVIEW

File: components/script/dom/webglrenderingcontext.rs [hunk 1]
```diff
@@ -2192,15 +2185,15 @@ impl WebGLRenderingContextMethods for WebGLRenderingContext {
     // https://www.khronos.org/registry/webgl/specs/latest/1.0/#5.14.3
     // FIXME: https://github.com/servo/servo/issues/20534
```
Comment (emilio) [inline components/script/dom/webglrenderingcontext.rs]: Remove the FIXME?

**Review by emilio — APPROVED** [review]
LGTM

**Comment by emilio** [comment]
Oh, I hadn't seen the comment above, and @nox asked me to review on IRC. Sorry for stealing the review :(

**Comment by nox** [comment]
@bors-servo r=emilio

I fixed the FIXMEs.

**Pull request merged**